.github/workflows/ci.yml

```diff
@@ -27,49 +29,60 @@ jobs:
   test:
     name: python${{ matrix.python-version }}-${{ matrix.platform.python-architecture }} ${{ matrix.platform.os }}
     runs-on: ${{ matrix.platform.os }}
+    needs: [lint, check-msrv, linalg-example]
     strategy:
-      max-parallel: 16
+      fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
Consider adding 3.10, and 3.11-dev (if numpy supports them?)

Reply:
I did, but it seems that this does not work without a Git dependency on PyO3 until 0.16 is released.